readme.md

```diff
@@ -1,65 +1,62 @@
-
-* [**Build a blockchain with Starport in a web-based IDE** (stable)](https://gitpod.io/#https://github.com/tendermint/starport/tree/master) or use [nightly version](https://gitpod.io/#https://github.com/tendermint/starport/)
-* [Check out the latest features in v0.19](https://starport.com/blog/starport-v0-19)
+[Ignite CLI](https://ignite.com/cli) is the all-in-one platform to build, launch, and maintain any crypto application on a sovereign and secured blockchain. It is a developer-friendly interface to the [Cosmos SDK](https://github.com/cosmos/cosmos-sdk), the world's most widely-used blockchain application framework. Ignite CLI generates boilerplate code for you, so you can focus on writing business logic.
 
 ## Quick start
 
-Open Starport [in your browser](https://gitpod.io/#https://github.com/tendermint/starport/tree/master), or [install it](https://docs.starport.network/guide/install.html). Create and start a blockchain:
+Open Ignite CLI [in your web browser](https://gitpod.io/#https://github.com/ignite-hq/cli/tree/master) (or open [nightly version](https://gitpod.io/#https://github.com/ignite-hq/cli/)), or [install latest release](https://docs.ignite.com/guide/install.html). Create and start a blockchain:
 
 ```bash
-starport scaffold chain github.com/cosmonaut/mars
+ignite scaffold chain github.com/cosmonaut/mars
 
 cd mars
 
-starport chain serve
+ignite chain serve
 ```
 
 ## Documentation
 
-To learn how to use Starport, check out the [Starport Documentation](https://docs.starport.com). To learn more about how to build blockchain apps with Starport, see the [Starport Developer Tutorials](https://docs.starport.com/guide/). 
+To learn how to use Ignite CLI, check out the [Ignite CLI docs](https://docs.ignite.com). To learn more about how to build blockchain apps with Ignite CLI, see the [Ignite CLI Developer Tutorials](https://docs.ignite.com/guide/). 
 
-To install Starport locally on GNU, Linux, or macOS, see [Install Starport](https://docs.starport.com/guide/install.html).
+To install Ignite CLI locally on GNU, Linux, or macOS, see [Install Ignite CLI](https://docs.ignite.com/guide/install.html).
 
-To learn more about building a JavaScript frontend for your Cosmos SDK blockchain, see [tendermint/vue](https://github.com/tendermint/vue).
+To learn more about building a JavaScript frontend for your Cosmos SDK blockchain, see [ignite-hq/vue](https://github.com/ignite-hq/vue).
 
 ## Questions
 
-For questions and support, join the official [Starport Discord](https://discord.gg/ignt) server. The issue list in this repo is exclusively for bug reports and feature requests.
+For questions and support, join the official [Ignite Discord](https://discord.gg/ignt) server. The issue list in this repo is exclusively for bug reports and feature requests.
```

Comment:
```suggestion
For questions and support, join the official [Ignite Discord](https://discord.gg/ignite) server. The issue list in this repo is exclusively for bug reports and feature requests.
```

Reply:
Hey @soniasingla, we had to merge the PR, can you please create another one with this change? Thank you.